Pages/Index.cshtml.cs

```diff
@@ -13,40 +13,69 @@ namespace NewYork_CIty_School_Data_With_Crime_Rate_History.Pages
 {
     public class IndexModel : PageModel
     {
+        public string GetData(string endpoint)
+        {
+            string downloadedData = "";
+            using (WebClient webClient = new WebClient())
+            {
+                downloadedData = webClient.DownloadString(endpoint);
+            }
+            return downloadedData;
+        }
```

Comment:
Added this GetData() method to avoid the duplication of code. This can be used whenever you need to pull any JsonData